CHANGELOG.md

```diff
@@ -18,18 +18,21 @@ NOTE: We are deprecating several collectors in this release.
 * [FEATURE] Add ZFS collector + review feedback from PRs 213 and 369 #410
 * [FEATURE] Add initial wifi collector #413
 * [FEATURE] Add NFS event metrics to mountstats collector #415
+* [FEATURE] Add an example rules file #422
 * [IMPROVEMENT] hwmon: Provide annotation metric to link chip sysfs paths to human-readable chip types #359
 * [IMPROVEMENT] Add node_filesystem_device_errors_total metric #374
 * [IMPROVEMENT] Add runit service dir flag #375
 * [IMPROVEMENT] Improve Docker documentation #376
 * [IMPROVEMENT] Ignore autofs filesystems on linux #384
 * [IMPROVEMENT] Replace some FreeBSD collectors with pure Go versions #385
 * [IMPROVEMENT] Use filename as label, move 'label' to own metric #411 (hwmon)
```

Comment:
Again, with the exception of this one where I annotated after the PR title, I'm literally copying the PR titles, nothing more.